### PR TITLE
chore: Dynamic tabs

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
- changes made in event and logs ui to show data in separate tabs rather than scrolling through data

- Current
image
image

- New
image
image

## Motivation and Context
- to make the logs more readable

## How did you test it?
- check out the screenshots attached above

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by JeevaRamu0104 — COMMENTED** [review]
image
image
Tab need to stick in the top

File: src/components/DynamicTabs.res [hunk 1]
```diff
@@ -375,11 +378,23 @@ let make = (
...
+
   let (collapsibleTabs, setCollapsibleTabs) = React.useState(_ => updatedCollapsableTabs)
   let (formattedOptions, setFormattedOptions) = React.useState(_ => [])
+
+  React.useEffect1(_ => {
+    setSelectedIndex(_ => initialIndex)
+    None
+  }, [initialIndex])
+
+  let _ = React.useMemo1(_ => {
```
Comment (sagarnaikjuspay) [inline src/components/DynamicTabs.res:394]: can u use useeffect

File: src/screens/Analytics/Logs/LogUtils/AuditLogUI.res [hunk 2]
```diff
@@ -10,7 +10,7 @@ module LogDetailsSection = {
       }
     }
 
-    <div className="border-b-2 border-border-light-grey pb-3 px-5 py-3">
+    <div className="border-border-light-grey pb-3 px-5 py-3">
```
Comment (sagarnaikjuspay) [inline src/screens/Analytics/Logs/LogUtils/AuditLogUI.res:13]: border-border-light-grey is this a right class name ?

File: src/screens/Analytics/Logs/LogUtils/AuditLogUI.res [hunk 3]
```diff
@@ -72,6 +120,36 @@ let make = (~id, ~urls, ~logType: LogTypes.pageType) => {
...
+    Some(activeTab)
+  }, [activeTab])
+
+  let setActiveTab = React.useMemo1(() => {
+    (str: string) => {
+      setActiveTab(_ => str->String.split(","))
+    }
+  }, [setActiveTab])
+
+  let _ = React.useMemo1(_ => {
```
Comment (sagarnaikjuspay) [inline src/screens/Analytics/Logs/LogUtils/AuditLogUI.res:149]: use useEffect

File: src/components/DynamicTabs.res (hunk 1 above)
Comment (gitanjli525) [reply src/components/DynamicTabs.res]: corrected

File: src/screens/Analytics/Logs/LogUtils/AuditLogUI.res (hunk 2 above)
Comment (gitanjli525) [reply src/screens/Analytics/Logs/LogUtils/AuditLogUI.res]: corrected

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**